Assets/Scripts/PlayerLife.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerLife : MonoBehaviour {
+
+    public void KillPlayer() {
```

Comment:
Why not throw this all in PlayerController?

Reply:
I'm thinking of getting rid of this script entirely since it doesn't really contribute much.  If we had a health system it might be a different story.  I'll probably just throw it on the Game Manager.

Reply:
Yeah, if this were a multi player game, I think this would make sense, but given there is only one player, it adds an unnecessary complication.